Packages/Netherlands3D/Core/Runtime/TileSystem/SelectSubObjects.cs

```diff
@@ -267,8 +284,10 @@ public void UnhideAll()
         IEnumerator FindSelectedSubObjectID(Ray ray, System.Action<string> callback)
         {
             //Check area that we clicked, and add the (heavy) mesh collider there
-            Vector3 planeHit = Vector3.zero; //CameraModeChanger.Instance.CurrentCameraControls.GetPointerPositionInWorld();
-            containerLayer.AddMeshColliders(planeHit);
+            Plane worldPlane = new Plane(Vector3.up, new Vector3(0, 0, 0));
```

Comment:
might become problematic if the terrainlevel is way above unity-y = 0 and looking at an angle. (thinking of limburg). yet, probalby good enough for now.